library/ssl_tls13_server.c

```diff
@@ -1442,11 +1442,6 @@ static int ssl_write_supported_version_ext( mbedtls_ssl_context *ssl,
     unsigned char *p = buf;
     *olen = 0;
 
-    if( ssl->handshake->extensions_present & SUPPORTED_VERSION_EXTENSION )
-    {
-        return( 0 );
-    }
-
```

Comment:
Couldn't the extension be missing here?

Reply:
The supported_version extension is a mandatory extension and it has to be sent.  Hence, it is not clear what this check is aiming to do.